.github/workflows/typescript-publish.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
             - name: Setup PNPM
               uses: pnpm/action-setup@v4
               with:
-                  version: 8
+                  version: 9.8.0 
```

Comment:
this has to be specified?

Reply:
it doesnt have to specify the fullpath but figured we might as well keep it in sync with whats in pacakge.json, which does have to specify the whole path